airbyte-tests/src/acceptanceTests/java/io/airbyte/test/acceptance/AcceptanceTests.java

```diff
@@ -1354,16 +1372,22 @@ private static ConnectionState waitForConnectionState(final AirbyteApiClient api
   }
 
   private static void waitForTemporalWorkflow(final UUID connectionId) {
-    /*
-     * do { try { Thread.sleep(1000); } catch (InterruptedException e) { throw new RuntimeException(e);
-     * } } while
-     * (temporalClient.isWorkflowRunning(temporalClient.getConnectionManagerName(connectionId)));
-     */
-    try {
-      Thread.sleep(10 * 1000);
-    } catch (InterruptedException e) {
-      throw new RuntimeException(e);
-    }
+    assertTimeout(Duration.ofSeconds(20), () -> {
```

Comment:
Should the create call actually block?

Reply:
The creation of the workflow can't return because of the fact that it wont't return (continue as new). Being able to perform a query is the way I have found to be able to check that the workflow is started properly.